src/DotNetNuke.PowerBI/scripts/ContentView.js

```diff
@@ -276,6 +290,26 @@
         this.pbifullscreen = function () {
             that.report.fullscreen();
         }
+
+        this.pbiedit = async function (hidePanes) {
+            await that.report.switchMode("edit");
+            if (hidePanes) {
+                const newSettings = {
+                    panes: {
+                        filters: {
```

Comment:
Filters bar should be visible depending on the Nav Filters setting (already in the module settings)